src/containers/FactoryLoader.tsx

```diff
@@ -84,21 +84,10 @@ export class FactoryLoaderContainer extends React.PureComponent<Props, State> {
   }
 
   private updateOverrideParams(key: string, val: string): void {
-    if (key.startsWith('override.')) {
-      const overrideKeys: string[] = [];
-      const pattern = new RegExp('([^.=]+)', 'g');
-      let regExpExecArray: RegExpExecArray | null = null;
-      while ((regExpExecArray = pattern.exec(key)) !== null) {
-        overrideKeys.push(regExpExecArray[0]);
-      }
-      if (overrideKeys.length > 0) {
-        let currentVal = this.overrideDevfileObject;
-        for (let index = 1; index < overrideKeys.length; index++) {
-          currentVal[overrideKeys[index]] = index === overrideKeys.length - 1 ? val : {};
-          currentVal = currentVal[overrideKeys[index]];
-        }
-      }
+    if (!key.startsWith('override.')) {
```

Comment:
Do you think it makes sense to keep this private func after you modified it to be really tiny? or better to inline if into https://github.com/eclipse-che/che-dashboard/blob/CHE-19424/src/containers/FactoryLoader.tsx#L220

I prefer a bit the second option

Reply:
I have fixed it

Reply:
https://github.com/eclipse-che/che-dashboard/pull/219/files#diff-4a8e21344754b4ee47f503546e5d899120cc756884f5b1c3b266399ddc3ec003R217